packages/jsapi-shim/README.md

```diff
@@ -1,43 +1,36 @@
-# @deephaven/log
+# @deephaven/jsapi-shim
 
-A logging library that can be used for modules to log different levels of logging, intercept console logging, and store a history of logs in memory for later consumption or exporting.
+A shim library that is used to import the global `dh` object as a module, and provide some useful `PropTypes`.
 
 ## Install
 
 ```bash
-npm install --save @deephaven/log
+npm install --save @deephaven/jsapi-shim
```

Comment:
Just FYI you don't need the save flag since npm5. It's the default in v5
https://github.com/npm/npm/issues/5108